data/json/items/gun/303.json

```diff
@@ -0,0 +1,57 @@
+[
+  {
+    "id": "smle_mk3",
+    "copy-from": "rifle_manual",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "Enfield battle rifle" },
+    "description": "An iconic weapon of both the First and Second World Wars, this bolt-action rifle has seen deployment in the hands of Commonwealth troops across Europe and beyond.  Outfitted with a 17-inch bayonet, firing the .303 calibre cartridge, and making use of 10-round Enfield magazines, the firearm’s reputation precedes it.",
+    "//": "Total unloaded weight of an Mk III is 3.96 kg, and a No. 4 is 4.11 kg.  total length of an Mk III is 1132mm, and a No. 4 is 1129mm.  Rounding to 4 kg of weight and 1130mm of length to meet in the middle.",
```

Comment:
please do not use the variant system for this. you have enough data to make two separate items

 if someone wants do even more reasearch  or just make a small change, you have to delete a bunch of stuff in the entry file, change a line in every itemgroup where the gun is used, add migration entries and probably some extra linting instead of simply changing a few numbers in a single json file. 

https://github.com/CleverRaven/Cataclysm-DDA/pull/61895/files here i had to change stuff in 5 different documents just to add an adjustable folding stock to the scar_l